thirdparty/libwebp/src/utils/bit_reader_utils.c

```diff
@@ -41,14 +41,7 @@ void VP8InitBitReader(VP8BitReader* const br,
   br->bits_    = -8;   // to load the very first 8bits
   br->eof_     = 0;
   VP8BitReaderSetBuffer(br, start, size);
-// -- GODOT -- begin
-#ifdef JAVASCRIPT_ENABLED // html5 required aligned reads
-  while(((uintptr_t)br->buf_ & 1) != 0 && !br->eof_)
-    VP8LoadFinalBytes(br);
-#else
```

Comment:
Seems like I didn't review that diff / the README comment well enough for this PR and removed this by mistake.

I haven't seen issues caused by the removal yet but I'll re-add these changes while updating to 1.2.2, and we can assess them again. @Faless